src/modules/App/App.js

```diff
@@ -7,33 +7,58 @@ import StyleSample from "../views/StyleSample";
 import DailyMission from "../views/DailyMission";
 import Error400 from "../common/error/Error400";
 import Error500 from "../common/error/Error500";
-import { ErrorBoundary } from "react-error-boundary";
 import AppContainer from "../../ui/containers/AppContainer";
+import { getAllMissions, getAllTasks, getAllUsers } from "../common/apiCalls";
 
-function App() {
+const App = () => {
   const [state, dispatch] = useReducer(appReducer, initialState);
+  const [error, setError] = useState(null);
```

Comment:
What do you think about placing these all inside one useEffect? Is there an advantage to three of them?